packages/e2e-test-utils/src/setAnalyticsCode.js

```diff
@@ -16,7 +16,7 @@
 /**
  * Internal dependencies
  */
-import { visitDashboard } from './';
+import { visitDashboard } from './index';
```

Comment:
can undo this change

Reply:
Sorry, my ide keeps this. https://github.com/google/web-stories-wp/pull/6275/commits/d8ba44562e453fd3a8f318f5d0028b8ea510bb41